.github/workflows/delete-artifacts.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   cleanup-docker:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
shouldn't this also be 22.04 so it doesnt get out of sync?

Reply:
No, this can be latest because the ubuntu version doesn't really matter for this workflow. This just runs an action that doesn't have any relation to our robot. The reason I set the other ones to a specific version is because they are responsible for building our images, and we don't want an updated version to break our image CI.